app/views/accounts/registrations/new.html.erb

```diff
@@ -1,42 +1,61 @@
+  <h3><%= f.label :country_code, "Country" %></h3>
+  <p id="togglenz"><%= f.country_select(:country_code, selected: @user.country_code || request.location.try(:country_code), priority_countries: ["NZ", "AU"]) %></p>
 
-  <p><%= f.label :school_graduation, "School Graduation" %><br />
+  <div class="nzonly">
+  <h3><%= f.label :school_graduation, "School Graduation (NZ Only)" %></h3>
+  <p> If you attend a primary, intermediate, or secondary school (or equivalent) in New Zealand then we ask that you please enter your estimated graduation date (the month and year you expect to finish secondary school by). You can update this later if it changes. Our website uses your graduation date to calculate your eligibility to:</p>
+  <ul>
+    <li>Compete <i>officially</i> in the New Zealand Informatics Competition (NZIC).</li>
+    <li>Potentially be invited to a programming camp run by the New Zealand Olympiad in Informatics (NZOI)</li>
+  </ul>
+  <p>Also note that:</p>
+  <ul>
+    <li>Anyone is welcome to create an account and use our site without specifying a graduation date.</li>
+    <li>Anyone can compete in the NZIC unofficially without specifying a graduation date. However, you will not be ranked on official scoreboards or be eligible to be invited to our programming camps.</li>
+    <li>Only New Zealand citizens or permanent residents that have attended camp will be considered to represent New Zealand at the International Olympiad in Informatics.</li>
+  </ul>
+  <p><b>Please select approximate secondary school graduation date or leave unspecified:</b></p>
   <%= f.fields_for :school_graduation do |graduation_form| %>
-  <%= graduation_form.radio_button('enabled', false, checked: !@user.school_graduation) %><%= f.label(:school_graduation_enabled_false, "Unspecified (Not considered eligible for IOI)") %> <br />
+  <%= graduation_form.radio_button('enabled', false, checked: !@user.school_graduation) %><%= f.label(:school_graduation_enabled_false, "Unspecified") %> <br />
   <%= graduation_form.radio_button('enabled', true, checked: !!@user.school_graduation) %><%= select_month(@user.school_graduation || 12, {prefix: 'user[school_graduation]'}, class: :radio_input) %><%= select_year(@user.school_graduation || Date.today, {start_year: [Date.today.year-50,@user.school_graduation.try(:year) || Date.today.year].min, end_year: Date.today.year+18, prefix: 'user[school_graduation]'}, class: :radio_input) %>
-  <% end %></p>
+  <% end %>
 
-  <p><%= f.label :school %><br />
+  <h3><%= f.label :school, "School (NZ Only)" %></h3>
   <%= f.fields_for :school, @user.school do |school_form| %>
     <%= school_form.text_field :name, class: [:school_select, :jsnodisplay] %>
-    <%= f.select :school_id, School.where(country_code: "NZ").order(:name).collect {|s| [s.name, s.id]}, { include_blank: true }, class: [:school_select, :jsdisplay], disabled: true %>, 
-    <%= school_form.country_select(:country_code, only: ["NZ"] ) %><br />
```

Comment:
Removing this causes new schools to be added with a nil `country_code`, so they won't appear under forms which use `School.where(country_code: "NZ")`.

Reply:
Should be fixed but someone else should probably review this.

Reply:
Hmmm. Good spot. A terrible way to solve this could be to add it back in but make it a hidden element on the page.

Reply:
If all schools are now only NZ schools then perhaps it is worth finding and replacing all instances of `School.where(country_code: "NZ")` and creating a db migration to drop this information. If so, I'd prefer to make this a separate pull request and reintroduce the field here so that this one doesn't get held up

Reply:
Sorry for not being clearer, but I already added a fix in c07c9f6. I just thought someone else should review my changes before we merge. c1378fa can be reverted.

Reply:
Oh, okay, sorry. In that case, do we also need to change the `school` method in the `User` class too?

Reply:
Clarification from @Holmes98 chat:

> If we remove the country code select box, POST requests will no longer contain the `user[school][country_code]` parameter (so it's treated as `nil`). Previously, this was used in the `school=` method to query for a school with that country code (the `first_or_initialize` part adds a new school if none was found). The fix is to just always use 'NZ' instead of that parameter.